web_view_leaflet_map/static/src/js/view/map/map_renderer.js

```diff
@@ -0,0 +1,31 @@
+odoo.define('web_view_leaflet_map.MapRenderer', function (require) {
+    'use strict';
+
+    var AbstractRenderer = require('web.AbstractRenderer');
+
+    var MapRenderer = AbstractRenderer.extend({
+        tagName: 'table',
+
+        _render: function () {
+            console.debug("MapRenderer:_render");
+            var $mainDiv = $("<div id='div_map' class='_INTRODUCED_IN_RENDER'>");
+            this.$el.append($mainDiv);
+
+            const container = document.getElementById('div_map')
```

Comment:
@legalsylvain could it be a missing semi-colon on L14?

Reply:
Hi, @brian10048. Thanks for your answer. I added the semi-colon, unfortunately, it doesn't fix the error.

in fact, the rest of the code is executed, but container is null.

See : log

![image](https://user-images.githubusercontent.com/3407482/190450301-6fde7b6f-0983-4918-8c16-46540e585ed0.png)

And result : 

![image](https://user-images.githubusercontent.com/3407482/190450996-a1ab24fa-63f5-4fc8-9b02-e4f8502fa7a9.png)

Reply:
I was just hoping for an easy answer!  Sorry I'm not really sure where to look next